gui-backend/src/main/java/com/rln/gui/backend/implementation/methods/AutoapproveApiImpl.java

```diff
@@ -29,40 +30,36 @@
 public class AutoapproveApiImpl {
 
   private static final Filter AUTO_APPROVE_MARKER_FILTER = new InclusiveFilter(
-    Set.of(AutoApproveTransferProposalMarker.TEMPLATE_ID), Map.of());
+      Set.of(AutoApproveTransferProposalMarker.TEMPLATE_ID), Map.of());
```

Comment:
Let's stick to 4 spaces in Java. https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent

Reply:
With Judy, we decided to use that formatter and set it up in IntelliJ, but sometimes (at least for me) it does not do its job.

Reply:
I was wrong. That very line was formatted by the Google Formatter plugin following the Google Style (as is, without changes).
![Screenshot from 2023-01-27 08-53-39](https://user-images.githubusercontent.com/33900822/215035790-7edf5b64-ea00-436e-b316-1eeb8ec79bb7.png)

Reply:
Because being unused, I just removed the line.